app/src/main/java/com/example/playlistmaker/ui/player/view_model/PlayerViewModel.kt

```diff
@@ -17,7 +17,7 @@ class PlayerViewModel(
     private val mediaInteractor: MediaInteractor,
     private val favoriteInteractor: FavoriteInteractor
 ): ViewModel(){
-    private val favouritLiveData = MutableLiveData<Boolean>()
+    val favouritLiveData = MutableLiveData<Boolean>()
```

Comment:
🍏 Раньше было правильнее:
```suggestion
    private val favouritLiveData = MutableLiveData<Boolean>()
```